.github/workflows/_ios-build-test.yml

```diff
@@ -121,15 +114,13 @@ jobs:
         if: inputs.ios-platform == 'SIMULATOR'
         run: |
           # shellcheck disable=SC1091
-          source "${RUNNER_TEMP}/anaconda/bin/activate"
+          source "${MINICONDA_INSTALL_PATH}/bin/activate" "${RUNNER_TEMP}/${CONDA_ENV}"
           # use the pytorch nightly build to generate models
           pip3 install --pre torch torchvision torchaudio -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
```

Comment:
For all of these commands to work, you would need to prefix them with `${CONDA_RUN}` to use the conda env you setup earlier

Reply:
`${CONDA_RUN}` prefix was added for the corresponding lines.